waku/node/v2/waku_types.nim

```diff
@@ -75,12 +72,18 @@ type
   HistoryPeer* = object
     peerInfo*: PeerInfo
 
+  MessageStoreResult*[T] = Result[T, string]
+
+  Sqlite* = ptr sqlite3
+
+  MessageStore* = ref object of RootObj
+    env*: Sqlite
```

Comment:
Can we make this accommodate for how things are currently being done with WakuMessages? That will significantly reduce risk of regression where we don't try to do three things at once in a PR.